torch/csrc/inductor/static_cuda_launcher.cpp

```diff
@@ -319,19 +349,41 @@ PyObject* launch_kernel(PyObject* self, PyObject* args) {
   CUfunction func = reinterpret_cast<CUfunction>(func_ptr); // NOLINT
   cudaStream_t cudaStream = reinterpret_cast<cudaStream_t>(stream); // NOLINT
   auto num_args = std::strlen(argTypes);
-  TORCH_CHECK(
-      num_args <= MAX_ARGS,
-      "Static Cuda Launcher only supports up to 120 arguments");
-  return launch_kernel_inner(
-      func,
-      gridX,
-      gridY,
-      gridZ,
-      numWarps,
-      sharedMemBytes,
-      argTypes,
-      varArgs,
-      cudaStream);
+  // Kernels with no arguments should just pass nullptr to cuLaunchKernel
+  if (num_args == 0) {
```

Comment:
Is this tested?

Reply:
Yup, by `test_incompatible_args` above the test I added. I will change the name of the test though, now that the arg is no longer incompatible